tests/functional/jobs/factory.py

```diff
@@ -165,6 +165,30 @@ def create_jobs(job_factory: JobFactory | None = None):
         },
     )
 
+    # Add a job with requested and allocated GPU to 0.
+    local_elapsed_time = elapsed_time * 1.5
+    job_factory.add_job(
+        job_id=999_999_999,
+        elapsed_time=local_elapsed_time,
+        start_time=end_time - timedelta(seconds=elapsed_time),
+        submit_time=end_time - timedelta(seconds=elapsed_time + 60),
+        allocated={
+            "billing": 2,
```

Comment:
```suggestion
            "billing": 14,
```
Pour cette job c'est le nombre de coeur le billing, puisqu'il n'y a pas de GPU.

Reply:
Fait ! https://github.com/mila-iqia/SARC/pull/93/commits/cfa1b85de9b726cac4fca6da8be5926e2ffff997